server/odc-service/src/main/java/com/oceanbase/odc/config/ScheduleConfiguration.java

```diff
@@ -195,7 +195,7 @@ public ThreadPoolTaskExecutor syncDatabaseTaskExecutor() {
         executor.setWaitForTasksToCompleteOnShutdown(true);
         executor.setAwaitTerminationSeconds(5);
         executor.setTaskDecorator(new TraceDecorator<>());
-        executor.setRejectedExecutionHandler(new ThreadPoolExecutor.DiscardPolicy());
```

Comment:
this pr #3667 has already fix this problem, we may cherry-pick it

Reply:
I feel the solution in pr #3667 is not the best solution. We submit the task in a for loop in the schedule, which is a fixed delay schedule. If we just make the queueSize to unlimited, the fixed delay schedule would actually  become a fixed rate schedule. If we set the rejected strategy to CallerRunsPolicy, this problem would be solved. @PeachThinking